rocAL/rocAL_pybind/examples/rocAL_api_pytorch_classification_reader.py

```diff
@@ -21,186 +21,81 @@
-        elif augmentation_name == "snp_noise":
-            output = fn.snp_noise(images)
-        elif augmentation_name == "snow":
-            output = fn.snow(images)
-        elif augmentation_name =="rain":
-            output = fn.rain(images)
-        elif augmentation_name == "fog":
-            output = fn.fog(images)
-        elif augmentation_name == "pixelate":
-            output = fn.pixelate(images)
-        elif augmentation_name == "exposure":
-            output = fn.exposure(images)
-        elif augmentation_name == "hue":
-            output = fn.hue(images)
-        elif augmentation_name == "saturation":
-            output = fn.saturation(images)
-        elif augmentation_name == "color_twist":
-            output = fn.color_twist(images)
-        elif augmentation_name == "crop_mirror_normalize":
-            output = fn.crop_mirror_normalize(images, device="cpu",
-                                              output_dtype=types.FLOAT16 if args.fp16 else types.FLOAT,
-                                              output_layout=types.NHWC if args.NHWC else types.NCHW,
-                                              crop=(300, 300),
-                                              image_type=types.RGB,
-                                              mean=[0, 0, 0],
-                                              std=[1, 1, 1])
-        elif augmentation_name == "nop":
-            output = fn.nop(images)
-        elif augmentation_name == "centre_crop":
-            output = fn.centre_crop(images)
-        elif augmentation_name == "color_temp":
-            output = fn.color_temp(images)
-        elif augmentation_name == "copy":
-            output = fn.copy(images)
-        elif augmentation_name == "rotate_fisheye_fog":
-            output1 = fn.rotate(images)
-            output2 = fn.fish_eye(output1)
-            output3 = fn.fog(output2)
-            pipe.set_outputs(output1, output2, output3)
-            output_set = 1
-        elif augmentation_name == "resize_brightness_jitter":
-            output1 = fn.resize(images, resize_x=300, resize_y=300)
-            output2 = fn.brightness(output1)
-            output3 = fn.jitter(output2)
-            pipe.set_outputs(output1, output2, output3)
-            output_set = 1
-        elif augmentation_name == "vignetter_blur":
-            output1 = fn.vignette(images)
-            output2 = fn.blur(output1)
-            pipe.set_outputs(output1, output2)
-            output_set = 1
-
-        if output_set==0:
-                pipe.set_outputs(output)
-    # build the pipeline
-    pipe.build()
-    # Dataloader
-    data_loader = ROCALClassificationIterator(pipe,device=device)
+    data_path = sys.argv[1]
+    if(sys.argv[2] == "cpu"):
+        _rali_cpu = True
+    else:
+        _rali_cpu = False
+    batch_size = int(sys.argv[3])
+    num_threads = 1
+    device_id = 0
+    random_seed = random.SystemRandom().randint(0, 2**32 - 1)
+
+    local_rank = 0
```

Comment:
Are we not having any unit testing for python now? If so mention that we will add it in future when we create the PR.

Reply:
No Shobana, this was a basic example provided with minimal support. we dont have a unit test yet.